website/project_styleguide/templates/patterns/molecules/streamfield/blocks/document_block.html

```diff
@@ -1,8 +1,17 @@
-<p class="document">
-
+<div class="media document clearfix">
     <a href="{{ value.document.url }}">
-        <svg height="40" viewBox="0 0 40 40" width="40" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="nonzero" stroke="#2f21f8"><path d="m12 22 8 8 8-8" stroke-width="2"/><circle cx="20" cy="20" r="19.5"/><path d="m20.0256348 9v21.0883789" stroke-width="2"/></g></svg>
-
-        {{ value.title|default:value.document.title }}
+        <i class="material-icons mx-2 display-3 text-primary">
```

Comment:
Generally speaking, we tend to advise against icon fonts. This isn't a good fit for all projects, but most are moving away from this approach and using svgs/sprites instead.

Reply:
Thanks! I agree on it, in next release we will try to get rid of these again - a quick fix for getting started.